docs/common/error-utilities.ts

```diff
@@ -405,9 +405,6 @@ const RENAMED_PAGES: Record<string, string> = {
   // Preview section
   '/preview/support/': '/preview/introduction/',
 
```

Comment:
Since the [React docs](https://react.dev/learn/react-compiler#usage-with-expo) forward to `/preview/react-compiler/`, I would set up the redirect the other way around.

```suggestion

'/preview/react-compiler/': '/guides/react-compiler/'
```

Reply:
Made a follow-up Pr to the React docs, thanks for pointing this out!